src/apps/AppCenter/DiscoverApps/DiscoverApps.js

```diff
@@ -1,121 +1,202 @@
+      <CardLayout columnWidthMin={columnWidthMin} rowHeight={rowHeight}>
+        {appsInDevelopment.map((app, i) => (
+          <AppCard key={i} app={app} compactMode={compactMode} />
+        ))}
+      </CardLayout>
+    </React.Fragment>
+  )
+})
 
-const AppsGrid = styled.div`
-  display: grid;
-  grid-auto-flow: row;
-  grid-gap: 25px;
-  justify-items: start;
-  grid-template-columns: 1fr;
-  ${breakpoint(
-    'medium',
-    `
-      grid-template-columns: repeat(auto-fill, 224px);
-    `
-  )};
-`
+function AppCard({ app, compactMode, onOpen, ...props }) {
+  const theme = useTheme()
+  const { link, icon, name, status, description } = app
+  const handleClick = useCallback(() => {
+    if (link) {
+      window.open(link, '_blank', 'noopener')
+    }
+  }, [link])
+
+  return (
+    <Card onClick={handleClick}>
+      <Main compactMode={compactMode}>
+        <StyledIconExternal
+          compactMode={compactMode}
+          theme={theme}
+          link={link}
+        />
+        <Icon compactMode={compactMode}>
+          <AppIcon size={9 * GU} src={icon} radius={12} />
+        </Icon>
+        <Name compactMode={compactMode}>{name}</Name>
+        <TagWrapper compactMode={compactMode} link={link}>
+          <Tag mode="indicator">{status}</Tag>
+        </TagWrapper>
+        <Description theme={theme} compactMode={compactMode}>
+          {description}
+        </Description>
+      </Main>
+    </Card>
+  )
+}
 
-const Main = styled(Card).attrs({ width: '100%', height: '288px' })`
+const Main = styled.section`
   ${unselectable};
   position: relative;
   overflow: hidden;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  padding-top: 25px;
+  height: 100%;
+  width: 100%;
+  white-space: initial;
+
+  ${({ compactMode }) =>
+    compactMode
+      ? `
+          display: grid;
+          grid-template-columns: auto 1fr auto;
+          grid-template-rows: auto auto auto;
+          grid-template-areas:
+            "empty topright"
+            "icon title"
+            "icon description";
+          padding: ${1.5 * GU}px ${1.5 * GU}px ${4 * GU}px ${3 * GU}px;
+        `
+      : `
+          display: flex;
+          flex-direction: column;
+          align-items: center;
+          padding-top: ${5 * GU}px;
+        `}
 `
 
 const Icon = styled.div`
-  height: 64px;
-  margin-bottom: 5px;
-  img {
-    display: block;
-  }
+  height: ${9 * GU}px;
+  width: ${9 * GU}px;
+  ${({ compactMode }) =>
+    compactMode
+      ? `
+        grid-area: icon;
+        margin-right: ${1.5 * GU}px;
+      `
+      : `
+        margin-bottom: ${2 * GU}px;
+      `}
 `
 
 const Name = styled.p`
   display: flex;
   width: 100%;
-  justify-content: center;
-  margin-bottom: 10px;
+  ${textStyle('title4')}
+  ${({ compactMode }) =>
+    compactMode
+      ? `
+        grid-area: title;
+        align-self: flex-end;
+      `
+      : `
+        justify-content: center;
+        margin-bottom: ${1 * GU}px;
```

Comment:
Super subjective, but perhaps one whole GU is a too much vertical spacing 😄; what do you think about just 0.25 or 0.5?